### PR TITLE
Static Analysis Doc Edit

### PR DESCRIPTION
**Pull request by alai97**
### What does this PR do? What is the motivation?

Chat with Kassen on Slack.

### Merge instructions

- [x] Please merge after reviewing

### CODE REVIEW

File: content/en/code_analysis/static_analysis/_index.md [hunk 1]
```diff
@@ -108,7 +108,30 @@ The two types of fixes are distinguished visually in the UI with different label
 {{< img src="code_analysis/static_analysis/static-analysis-ai-fix.png" alt="Visual indicator of an AI static analysis suggested fix" style="width:60%;">}}
 
 ### Ignoring violations
-To ignore a rule globally or by path(s), see [Static Analysis Setup][5]. 
+
+#### Ignore for a repository
+Add an ignore rule in your `static-analysis.datadog.yml` file. This example below ignores the rule `javascript-express/reduce-server-fingerprinting` for all directories.
```
Comment (buraizu) [inline content/en/code_analysis/static_analysis/_index.md:113]:
```suggestion
Add an ignore rule in your `static-analysis.datadog.yml` file. The example below ignores the rule `javascript-express/reduce-server-fingerprinting` for all directories.
```

File: content/en/code_analysis/static_analysis/_index.md [hunk 2]
```diff
@@ -108,7 +108,30 @@ The two types of fixes are distinguished visually in the UI with different label
...
+```
+rulesets:
+  - javascript-express
+    rules:
+      reduce-server-fingerprinting:
+        ignore: "**"
+```
+
+#### Ignore for a file or directory
+Add an ignore rule in your `static-analysis.datadog.yml` file. The example below ignores the rule `javascript-express/reduce-server-fingerprinting` for this file. For more information on how to ignore by path, see [Static Analysis Setup][5]. You can adapt this to use a path (or see our documentation about our configuration file).
```
Comment (buraizu) [inline content/en/code_analysis/static_analysis/_index.md:124]:
```suggestion
Add an ignore rule in your `static-analysis.datadog.yml` file. The example below ignores the rule `javascript-express/reduce-server-fingerprinting` for this file. For more information on how to ignore by path, see [Static Analysis Setup][5]. You can adapt this to use a path (or see the configuration file documentation).
```

Comment (buraizu) [reply content/en/code_analysis/static_analysis/_index.md:124]: Minor update per the style guide here. Would it also be possible to provide a link to the configuration file documentation?

**Review by buraizu — APPROVED** [review]
Thanks, approving with some minor feedback and a question

File: content/en/code_analysis/static_analysis/_index.md (hunk 2 above)
Comment (alai97) [reply content/en/code_analysis/static_analysis/_index.md:124]: The last sentence is repetitive, so we can remove it. Path prefixes are covered in the Setup doc page :)

**Pull request merged**